swift_scality_backend/diskfile.py

```diff
@@ -192,6 +193,11 @@ def zero_copy_send(self, wsockfd):
 
             try:
                 conn.putrequest('GET', object_url.path, skip_host=False)
+                if client._url_username and client._url_password:
+                    creds_str = ('%s:%s' % (client._url_username, client._url_password))
+                    basic_auth_header = get_urllib3().util.make_headers(basic_auth=creds_str)
```

Comment:
Can you keep a reference to get_urllib3().util.make_headers in a fixed location? You would avoid avoid to do it for every request.
NB: my comment is merely because perhaps calling get_urllib3() is not 'free'

Reply:
Fixed!